prediction_market_agent/agents/think_thoroughly_agent/think_thoroughly_agent.py

```diff
@@ -472,25 +485,23 @@ def generate_prediction_for_one_outcome(
             )
 
         api_keys = APIKeys()
-
+        agent = ThinkThoroughlyWithPredictionProphetResearch.build_pydantic_ai_agent(
+            model=model
+        )
         research = prophet_research(
             goal=scenario,
             initial_subqueries_limit=0,  # This agent is making his own subqueries, so we don't need to generate another ones in the research part.
             max_results_per_search=5,
             min_scraped_sites=2,
-            agent=PydanticAIAgent(
-                model=model, model_settings=ModelSettings(temperature=0.7)
-            ),
+            agent=agent,
             openai_api_key=api_keys.openai_api_key,
             tavily_api_key=api_keys.tavily_api_key,
         )
+
         prediction = prophet_make_prediction(
             market_question=scenario,
             additional_information=research.report,
-            agent=PydanticAIAgent(
-                model=model,
-                model_settings=ModelSettings(temperature=LLM_SUPER_LOW_TEMPERATURE),
-            ),
+            agent=agent,
```

Comment:
By using the same agent class, you changed the temperature used for `make_prediction`, is that an accident or is there a reason for that?

Reply:
Maybe there was a typo, now I see it's using the same temperature (LLM_SUPER_LOW_TEMPERATURE) as main